contract/contract.go

```diff
@@ -323,10 +315,8 @@ func preloadWorker() {
 		}
 
 		// create a new context
-		ctx := NewVmContext(bs, nil, nil, receiver, contractState, txBody.GetAccount(),
-			tx.GetHash(), request.bi, "", false, false, receiver.RP(),
-			request.preloadService, txBody.GetAmountBigInt(), txBody.GetGasLimit(),
-			txBody.Type == types.TxType_FEEDELEGATION)
+		// FIXME need valid context
+		ctx := NewVmContext(context.Background(), bs, nil, nil, receiver, contractState, txBody.GetAccount(), tx.GetHash(), request.bi, "", false, false, receiver.RP(), request.preloadService, txBody.GetAmountBigInt(), txBody.GetGasLimit(), txBody.Type == types.TxType_FEEDELEGATION)
```

Comment:
Can it be merged without the "valid context"?
If yes, maybe the FIXME can be removed?